### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by soGaussian**
I added a config file to segment the users by paid vs organic. Kindly let me know if I did it right, first time setting up a config file

### CODE REVIEW

File: jetstream/lifestyles-images-onboarding-experiment-v3.toml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+## EXPERIMENT SPECIFIC
+[experiment]
+segments = ['attributed', 'non_attributed']
+
+start_date = "2023-06-14"
+
+end_date = "2023-06-27"
```
Comment (bani) [inline jetstream/lifestyles-images-onboarding-experiment-v3.toml:7]: What are the actual dates for start, end of enrolment, and end of observation period or the experiment? Two weeks total seems extremely short.

File: jetstream/lifestyles-images-onboarding-experiment-v3.toml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+[segments.non_attributed]
+select_expression = 'max(eligible_segment = FALSE)'
+data_source = "lst"
+
+## Data Sources
+[segments.data_sources.lst]
+from_expression = """
+    (SELECT a.*
+      , CASE WHEN b.adjust_network in ("Google Ads ACI", "Product Marketing (Owned media)") THEN TRUE else FALSE END AS eligible_segment
```
Comment (bani) [inline jetstream/lifestyles-images-onboarding-experiment-v3.toml:30]: I'd normally leave the CASE out if the returned value should be TRUE/FALSE.
Writing just `b.adjust_network in ("Google Ads ACI", "Product Marketing (Owned media)") AS eligible_segment` achieves the same result and I think it's more legible

File: jetstream/lifestyles-images-onboarding-experiment-v3.toml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        CROSS JOIN UNNEST(ping_info.experiments) AS event
+        WHERE
+            DATE(submission_timestamp) >= '2023-06-09'
+          AND event.key = "lifestyles-images-onboarding-experiment-v3"
+       GROUP BY  1, 2)  a
+    LEFT JOIN (
+            SELECT
+                    client_id
+                    , adjust_network
+            FROM moz-fx-data-shared-prod.fenix.firefox_android_clients
```
Comment (bani) [inline jetstream/lifestyles-images-onboarding-experiment-v3.toml:46]: why are you looking at both the first_session ping and the firefox_android_clients table?

File: jetstream/lifestyles-images-onboarding-experiment-v3.toml [hunk 4]
```diff
@@ -0,0 +1,39 @@
+## EXPERIMENT SPECIFIC
+[experiment]
+segments = ['attributed', 'non_attributed']
+
+start_date = "2023-06-14"
+
+enrollment_period = 14
+
+end_date = "2023-07-267"
```
Comment (danielkberry) [inline jetstream/lifestyles-images-onboarding-experiment-v3.toml:9]: Should this be `"2023-07-26"` or  `"2023-07-27"`?

File: jetstream/lifestyles-images-onboarding-experiment-v3.toml (hunk 3 above)
Comment (soGaussian) [reply jetstream/lifestyles-images-onboarding-experiment-v3.toml]: I made the decision to add first session ping to align with the definition of new profiles. The android clients table presents the cleanest way for me to segment based on paid and non paid using the adjust network. I am open to use other tables you recommend if this table and approach is flawed

File: jetstream/lifestyles-images-onboarding-experiment-v3.toml (hunk 1 above)
Comment (soGaussian) [reply jetstream/lifestyles-images-onboarding-experiment-v3.toml]: I have fixed it

**Review by bani — APPROVED** [review]

File: jetstream/lifestyles-images-onboarding-experiment-v3.toml [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+
+
+## SEGMENTS
+# A segment for users that were acquired through ctd creatives
+# first run feature.
+[segments]
+
+[segments.attributed]
+select_expression = 'max(eligible_segment = TRUE)'
```
Comment (danielkberry) [inline jetstream/lifestyles-images-onboarding-experiment-v3.toml:21]: You might want to do `COALESCE(MAX(eligible_segment = TRUE), FALSE)` here to protect against nulls. If a user doesn't report a first session ping, they might have a null value here and this explicitly keeps those users out of this segment.

**Review by danielkberry — APPROVED** [review]
Other than that comment on nulls, LGTM

**Pull request merged**